dental-office.scss

```diff
@@ -6,4 +6,25 @@ body {
     padding: 0 20px; 
 }
 
+// Index 
```

Comment:
Nice work on adding comments, however I'm not sure what the '// Index' comment specifies. Maybe change it to something like // Header Section or just // Header. Something along those lines would be great. Other than that good job, keep up the great work!!!